docs/scripts/uploadBenchmark.py

```diff
@@ -380,8 +464,10 @@ def execute_test_suite(
     # execute_sync_to_s3(path, test_name)
 
 
-syn = synapseclient.Synapse(debug=False)
+syn = synapseclient.Synapse(debug=True, http_timeout_seconds=600)
```

Comment:
Since we found that a higher default `http_timeout_seconds` as a bit more stable, is that a case to raise the defaults to a higher value @linglp ?